04_numeronym/kasunfdo/main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func numeronyms(s ...string) []string {
+	out := make([]string, len(s))
+	for i, str := range s {
+		out[i] += getNumeronym(str)
+	}
+	return out
+}
+
+func getNumeronym(str string) string {
+	trimmedString := strings.TrimSpace(str)
+
+	//  Unicode string could not be accessed using index
+	//  Therefore, string is converted into an array of runes
+	runeArr := []rune(trimmedString)
```

Comment:
Variable names in Go are not usually named with the type (`Arr` in this case) (https://dave.cheney.net/practical-go/presentations/qcon-china.html#_dont_name_your_variables_for_their_types). Also, this is a slice not an array so this is even misleading.

I would just call this `runes`.